freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/user/doc/UserModelDescriptions.java

```diff
@@ -6,6 +6,7 @@
     public static final String USERSYNC_ENVIRONMENT_CRNS = "Optional environment crns to sync";
     public static final String USERSYNC_USER_CRNS = "Optional user crns to sync";
     public static final String USERSYNC_MACHINEUSER_CRNS = "Optional machine user crns to sync";
+    public static final String DELETED_WORKLOAD_USER = "An Optional workload user to be deleted";
```

Comment:
I think we want to be clear that this is a workload username, not a user crn, username, etc.
We should also indicate that this workload username corresponds to the single user or machine user crn in the filters.

Reply:
Okay, I've updated it. Since we're planning on moving to newer endpoints soon, I wasn't worrying too much about usability here.